CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Ignite UI for Angular Change Log
 
 All notable changes for each version of this project will be documented in this file.
+## 7.3.12
+ - `IgxDatePicker` changes
+    - `onValueChanged` event is added.
```

Comment:
I'd make this `valueChange` to match the `value` Input so it can be [two-way bindable](https://angular.io/guide/template-syntax#two-way-binding-). Unless there's a chance we'd need a cancellable `valueChanging` at some point.

_I realize that'd be inconsistent with the Time Picker (😞), but I see no reason to have this specific event and skip on the extra functionality. Perhaps we can update it as well in the future._